test/connector/tcp/mssql/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mcr.microsoft.com/mssql-tools:latest
+
+ENV PATH $PATH:/opt/mssql-tools/bin
+
+# Install wget. But before we do the required `apt-get update`, we first
+# remove the Microsoft package distro list that was created in the base
+# image, so that we're not subject to connection/checksum issues in that
+# distro (as we've bitten by in the past).
```

Comment:
Could you add a little more context?  I'm still a little hazy on exactly what happened.  I will definitely have no idea when I read this in 2 months.

Reply:
I reworded the comment, adding more details.